ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml

```diff
@@ -3150,3 +3150,116 @@ periodics:
     - name: manifest-tool-local-pusher
       secret:
         secretName: manifest-tool-local-pusher
+- agent: kubernetes
+  cluster: build05
+  cron: '@yearly'
+  decorate: true
+  labels:
+    config-type: modern
+    job-architecture: multi
+    job-env: hypershift-hosted-powervs
+    job-type: launch
+  name: release-openshift-origin-installer-launch-hypershift-hosted-powervs
+  spec:
+    containers:
+    - args:
+      - --gcs-upload-secret=/secrets/gcs/service-account.json
+      - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+      - --lease-server-credentials-file=/etc/boskos/credentials
+      - --report-credentials-file=/etc/report/credentials
+      - --secret-dir=/secrets/ci-pull-credentials
+      - --secret-dir=/usr/local/launch-cluster-profile
+      - --input-hash=$(BUILD_ID)
+      - --input-hash=$(JOB_NAME)
+      - --namespace=$(NAMESPACE)
+      - --target=launch
+      command:
+      - ci-operator
+      env:
+      - name: CLUSTER_DURATION
+        value: "9000"
+      - name: BRANCH
+        value: "4.14"
+      - name: JOB_NAME_SAFE
+        value: launch
+      - name: UNRESOLVED_CONFIG
+        value: |
+          releases:
+            multi-latest:
+              candidate:
+                architecture: multi
+                product: ocp
+                stream: nightly
+                version: "4.14"
+          resources:
+            '*':
+              limits:
+                memory: 4Gi
+              requests:
+                cpu: 100m
+                memory: 200Mi
+          tests:
```

Comment:
Might need to add RELEASE_IMAGE_LATEST dep pointing to release:multi-latest like you have added in hypershift periodics.